resources/lang/en/messages.php

```diff
@@ -23,4 +23,5 @@
     'num_unique'      => 'This number must be unique in the data table.',
     'closed_search'   => 'This investigation is currently closed.',
     'closed_practice' => 'This practice is currently closed.',
+    'no_groups'       => 'No groups have been registered',
```

Comment:
A dot in the end is missing. 

'no_groups'       => 'No groups have been registered.',

Reply:
ok